go/adbc/driver/snowflake/driver.go

```diff
@@ -67,6 +67,13 @@ const (
 	// "300ms", "1.5s" or "1m30s". ParseDuration accepts negative values
 	// but the absolute value will be used.
 	OptionClientTimeout = "adbc.snowflake.sql.client_option.client_timeout"
+	// OptionUseHighPrecision controls the data type used for NUMBER columns
+	// using a FIXED size data type. By default, this is enabled and NUMBER
+	// columns will be returned as Decimal128 types using the indicated
+	// precision and scale of the type. If disabled, then fixed-point data
+	// with a scale of 0 will be returned as Int64 columns, and a non-zero
+	// scale will return a Float64 column.
+	OptionUseHighPrecision = "adbc.snowflake.sql.client_option.use_high_precision"
```

Comment:
nit: can we also add this to the Python enum?

Reply:
added, also updated the docs in general with the new option.